testsuite/module-publish/src/packages/simple/sources/Simple.move

```diff
@@ -411,6 +412,25 @@ module 0xABCD::simple {
         }
     }
 
+    struct TableStore has key {
+        table_entries: Table<u64, u64>
+    }
+
+    fun make_or_change_table(owner: &signer, count: u64) acquires TableStore {
```

Comment:
this always changes the same set of elements. 
maybe more flexible is to also pass offset here, and then allow changing different sub-interval of the table if so desired.

You can pass 0 for offset in your MakeOrChange entry point, but someone can add another more flexible entry point if they want to (i.e. we could have a MakeOrChangeRandom(max_count, max_offset), which randomly select count of keys and offset based on those max values.

Reply:
Done.